package.json

```diff
@@ -1152,7 +1218,7 @@
         "compile": "webpack --mode development && npm run buildScripts",
         "recompile": "npm run clean && npm run compile",
         "watch": "npm run buildScripts && tsc -watch -p ./",
-        "postinstall": "npm run installSSMTemp && ts-node ./build-scripts/generateServiceClient.ts && npm run generateTelemetry && npm run generateConfigurationAttributes",
+        "postinstall": "ts-node ./build-scripts/generateServiceClient.ts && npm run generateTelemetry && npm run generateConfigurationAttributes",
```

Comment:
Did you try reverting this change to see if that fixes the integ test hang? 

Locally, try this:

```
rm -rf dist
npm run testCompile
NO_COVERAGE=true npm run integrationTest
```